src/app/dataset-view/additional-components/metadata-component/components/add-polling-source/add-polling-source.component.ts

```diff
@@ -109,8 +109,8 @@ export class AddPollingSourceComponent extends BaseMainEventComponent implements
         this.subsribeErrorMessage();
     }
 
-    public changeStep(step: SetPollingSourceSection): void {
-        this.currentStep = step;
+    public changeStep(step: SetPollingSourceSection | AddPushSourceSection): void {
```

Comment:
Make a type definition. This is used in many places

Reply:
Created  `type SourcesSection=SetPollingSourceSection | AddPushSourceSection`